adapters/madvertise/madvertisetest/supplemental/required-zoneid-.json

```diff
@@ -0,0 +1,22 @@
+{
+  "mockBidRequest": {
```

Comment:
Typo: Looks like there's typo in the name of this file. There's an extra hyphen in the end. Can you please remove that and rename the file?

Reply:
done. Sorry
[master 3c2e399f] remove extra hyphen in the end.